services/vercel.js

```diff
@@ -2,11 +2,11 @@ const axios = require("axios");
 
 module.exports = {
   async getDeployments() {
-    const res = await this._getClient().get(`/v5/now/deployments?projectId=${this._getApiProjectId()}&teamId=${this._getApiTeamId()}&limit=10`);
+    const res = await this._getClient().get(`/v6/now/deployments?projectId=${this._getApiProjectId()}&limit=10`);
     return res.data;
   },
   async getDeployment(id) {
-    const res = await this._getClient().get(`/v11/now/deployments/${id}?projectId=${this._getApiProjectId()}&teamId=${this._getApiTeamId()}`);
```

Comment:
Current version of this endpoint is v11, according to https://vercel.com/docs/api#endpoints/deployments/get-a-single-deployment.

Did you try it with your change and v11?

Reply:
Yeah v11 didn't return anything. I checked my dev tools while browsing to the appropriate place in their webapp and there were multiple requests actually with the deploymentId, `v5` and `v7` I believe. One of them didn't work here so I went with the other that did return all the necessary data in the correct shape.

Reply:
Just double checked, seems to be `v10` and `v13` they're using atm.